compiler-rt/lib/asan/asan_fake_stack.cpp

```diff
@@ -55,17 +55,32 @@ FakeStack *FakeStack::Create(uptr stack_size_log) {
   if (stack_size_log > kMaxStackSizeLog)
     stack_size_log = kMaxStackSizeLog;
   uptr size = RequiredSize(stack_size_log);
+  uptr padded_size = size + (1 << kMaxStackFrameSizeLog);
```

Comment:
I think this would be a bit easier to read if we had

```
constexpr auto kMaxStackFrameSize = 1 << kMaxStackFrameSizeLog;
```

and then used this

Reply:
Added a static const